### PR TITLE
Add comment describing why -ia is not the alias for -InformationAction common parameter

### LINKED ISSUE #10237: The -InformationAction common parameter alias is inconsistent with other stream common parameter aliases
All common parameters that have to do with streams follow a two-letter alias format (first letter of the stream name followed by the first letter of "Action" or "Variable"). In fact, `-InformationVariable` has an alias of `iv`. So why does `-InformationAction` break the mold with an alias of `infa` instead of simply using `ia` as an alias? I couldn't find any competing `ia` alias in common parameters.

This issue should be addressed by adding `ia` as a secondary alias for the `-InformationAction` common parameter.

### PR DESCRIPTION
**Pull request by KirkMunro**
# PR Summary

This adds a comment highlighting why a common parameter alias name breaks the naming convention used for other common parameters.

## PR Context

This comment is being added so that others looking at this code in the future will know why the alias naming convention is different in this case.

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [X] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [X] Not Applicable
    - **OR**
    - [] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [] Issue filed:
- **Testing - New and feature**
    - [X] N/A or can only be tested interactively
    - **OR**
    - [] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [X] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: src/System.Management.Automation/engine/CommonCommandParameters.cs [hunk 1]
```diff
@@ -109,7 +109,7 @@ public ActionPreference WarningAction
         /// This parameter tells the command what to do when an informational record occurs.
```
Comment (PoshChan) [inline src/System.Management.Automation/engine/CommonCommandParameters.cs]: @KirkMunro, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/demos/DSC/readme.md.https://docs.microsoft.com/powershell/dsc/overview should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```

Comment (KirkMunro) [reply src/System.Management.Automation/engine/CommonCommandParameters.cs]: @PoshChan Please restart static analysis

**Comment by daxian-dbw** [comment]
There is a possibility of breaking existing scripts with parameters that has `ia` alias or starts with `ia`.
Maybe no one knows anymore, but I wonder maybe `infa` was intentionally chosen at the time to avoid a breaking change.

**Comment by KirkMunro** [comment]
@daxian-dbw I thought about that, and did a search across all commands I have (native + installed modules like Azure, AWS, etc.) and across 9870 cmdlets there aren't any parameters that have `ia` as an alias. I think it might just be something that was missed, because we don't Pester test parameter aliases.

**Comment by daxian-dbw** [comment]
@PowerShell/powershell-committee Please review the proposed new alias `-ia` for the common parameter `-InformationAction`.
There is a possibility of breaking existing scripts with parameters that has `ia` alias or starts with `ia`, but also it might fall into the bucket 3 breaking change.

**Comment by SteveL-MSFT** [comment]
@PowerShell/powershell-committee reviewed this, since an explicit alias was chosen when Information Stream feature was added we investigated if there is a conflict with existing use of `ia` as a parameter alias.  We found one instance with `Find-UpgradeableChocolateyPackage` that has `ia` as alias for `-InstallArgs`.  For this reason, we don't accept the proposal for `ia` alias for `InformationAction`.

**Comment by SteveL-MSFT** [comment]
Thanks to @LeeHolmes who dug through ancient history, originally this parameter did have the `ia` alias, but it caused a breaking change as the inbox cmdlet [New-IscsiTargetPortal](https://docs.microsoft.com/en-us/powershell/module/iscsi/new-iscsitargetportal?view=win10-ps) was already using that alias.  So it was changed to `infa`.

**Comment by KirkMunro** [comment]
@SteveL-MSFT Thanks to both of you.

Not that this comes up often, but it would be worth having a comment in the code that points this out, no? I can always change this PR to just add that, or close this and submit another PR.

**Comment by SteveL-MSFT** [comment]
@KirkMunro please update the PR to add a comment on why it's not `ia` so we don't have to dig through history again in the future :)

**Comment by KirkMunro** [comment]
Done.

Aside: If tab completion used some fuzzy logic to match alias names, I think most parameter aliases wouldn't be necessary.

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by SteveL-MSFT** [comment]
@PoshChan please retry macos

**Comment by PoshChan** [comment]
@SteveL-MSFT, successfully started retry of `PowerShell-CI-macOS`

**Pull request merged**